examples/compiled/histogram_bin_transform.vg.json

```diff
@@ -14,15 +14,17 @@
         {
           "type": "extent",
           "field": "IMDB_Rating",
-          "signal": "bin_maxbins_10_IMDB_Rating_extent"
+          "signal": "bin_maxbins_10_IMDB_Rating_bin_IMDB_Rating_bin_IMDB_Rating_end_extent"
```

Comment:
This signal looks nightmarish, so I'm not convinced that the output name should be in the signal.  

In fact, if there are two similar bin transforms (with different `as`), wouldn't it be better to create only one extent transform and one bin transform to avoid redundant computation. We then instead derive new fields (with Vega formula) according to multiple "as".

Reply:
I imagine refactoring `as` in the bin node to be `[string, string][]` instead of `[string, string]` and allow merging non-unique "as" tuple.

Reply:
That seems like a sensible solution to me. I'll try getting that working.

Reply:
Thanks — sorry I didn’t see Dom’s comment earlier.

Reply:
Thanks — sorry I didn’t see Dom’s comment earlier.

Reply:
Let’s close  this PR and open another one then since it will be a totally different solution